bigquery/routine_test.go

```diff
@@ -72,9 +72,16 @@ func TestRoutineTypeConversions(t *testing.T) {
 		in         interface{}
 		want       interface{}
 	}{
-		{"empty", "ToRoutineMetadata", &bq.Routine{}, &RoutineMetadata{}},
-		{"basic", "ToRoutineMetadata",
-			&bq.Routine{
+		{
```

Comment:
It looks like the method `RoutineMetadata.toBQ()` isn't tested. I wonder if we need to add unit tests for it.

Reply:
added